pywbemtools/pywbemcli/_context_obj.py

```diff
@@ -281,7 +281,7 @@ def execute_cmd(self, cmd):
             self.spinner_start()
         if self.pdb:
             import pdb  # pylint: disable=import-outside-toplevel
-            pdb.set_trace()
+            pdb.set_trace()  # pylint: disable=no-member
```

Comment:
In my 3.8 environment, this Pylint issue does not surface. Its version is the latest for Python 3.8:
```
$ pylint --version
pylint 2.4.4
astroid 2.3.3
Python 3.8.5 (default, Jul 21 2020, 10:48:26) 
[Clang 11.0.3 (clang-1103.0.32.62)]
```

Can you please post the Pylint version from the environment where Pylint reports the the "no-member" issues?

Reply:
I am using python 3.6.7 and it always shows up.  The error is that it says set_trace attribute does not exist.

Reply:
pylint 2.4.4

Reply:
Very strange. On Python 3.6.7 on Ubuntu in Travis (with pylint 2.4.4), the issue does not show up: https://travis-ci.org/github/pywbem/pywbemtools/jobs/716348761#L1749

Could you please recreate your Python virtualenv and run `make pylint`? It should install all required dependencies that way.

Reply:
I concluded that this is an issue in the particular python environment and it may be because the current python3 installed is 3.6.9 (what occurs on new virtual environment created) but the virtual environment that I normally use reports python 3.6.8.
Looks like ubuntu updated minor version recently.

Removing the pylint disable statements.